browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -2407,20 +2393,80 @@ index a166bcac55d10d7f1d758eb43fc091136e8865ad..af023d66b56b3f68d6b7aa0be6151b43
  		F12171F316A8BC63000053CA /* WebVTTElement.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = WebVTTElement.cpp; sourceTree = "<group>"; };
  		F12171F416A8BC63000053CA /* WebVTTElement.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = WebVTTElement.h; sourceTree = "<group>"; };
  		F32BDCD52363AAC90073B6AE /* UserGestureEmulationScope.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = UserGestureEmulationScope.cpp; sourceTree = "<group>"; };
-@@ -24353,7 +24368,12 @@
- 				1AF326770D78B9440068F0C4 /* EditorClient.h */,
- 				E36D701E27B71F04006531B7 /* EmptyAttachmentElementClient.h */,
- 				93C09A800B064F00005ABD4D /* EventHandler.cpp */,
+@@ -24624,6 +24639,11 @@
+ 				BC4A5324256055590028C592 /* TextDirectionSubmenuInclusionBehavior.h */,
+ 				2D4F96F11A1ECC240098BF88 /* TextIndicator.cpp */,
+ 				2D4F96F21A1ECC240098BF88 /* TextIndicator.h */,
++				F050E16923AD660C0011CE47 /* Touch.cpp */,
 +				F050E16F23AD669E0011CE47 /* TouchEvent.cpp */,
 +				F050E17023AD669F0011CE47 /* TouchEvent.h */,
- 				93C09A520B064DB3005ABD4D /* EventHandler.h */,
-+				F050E16923AD660C0011CE47 /* Touch.cpp */,
 +				F050E16C23AD66630011CE47 /* TouchList.cpp */,
 +				F050E16B23AD66620011CE47 /* TouchList.h */,
- 				E0FEF371B27C53EAC1C1FBEE /* EventSource.cpp */,
- 				E0FEF371B17C53EAC1C1FBEE /* EventSource.h */,
- 				E0FEF371B07C53EAC1C1FBEE /* EventSource.idl */,
-@@ -30406,6 +30426,8 @@
+ 				F48570A42644C76D00C05F71 /* TranslationContextMenuInfo.h */,
+ 				F4E1965F21F26E4E00285078 /* UndoItem.cpp */,
+ 				2ECDBAD521D8906300F00ECD /* UndoItem.h */,
+@@ -24765,8 +24785,6 @@
+ 		6FB7D2D5250FD7B5000207AA /* flex */ = {
+ 			isa = PBXGroup;
+ 			children = (
+-				6F047A9128453EDB00C25EE7 /* FlexLayout.h */,
+-				6F047A9028453ED200C25EE7 /* FlexLayout.cpp */,
+ 				6FB17475283A8FF40067D8CA /* FlexFormattingConstraints.h */,
+ 				6FB7D2D7250FD7E5000207AA /* FlexFormattingContext.cpp */,
+ 				6FB7D2D9250FD7FC000207AA /* FlexFormattingContext.h */,
+@@ -24774,6 +24792,8 @@
+ 				6FF911F626487FC8002021DF /* FlexFormattingGeometry.h */,
+ 				6FB7D2D8250FD7EF000207AA /* FlexFormattingState.cpp */,
+ 				6FB7D2DA250FD7FC000207AA /* FlexFormattingState.h */,
++				6F047A9028453ED200C25EE7 /* FlexLayout.cpp */,
```

Comment:
Were there some conflicts in the xcode project files?

Reply:
I built in Mac and everything went find. I noticed that when I run `prepare_checkout.sh` the script `Tools/Scripts/sort-Xcode-project-file` was called too (I noticed this because I got one error the script could not be found in one of the containers I used). I think this is the reason for the changes in `webkit/embedder/Playwright/Playwright.xcodeproj/project.pbxproj`, what changed is that now the list of files is sorted.

Reply:
I see, let's land it with the project file changes. It's still unclear where the whitespace changes come from, but let's land it as is, should be safe too.

Reply:
You need to rebase though as there was another commit in playwright touching bootstrap.diff